package.json

```diff
@@ -25,10 +25,12 @@
     "preinstall": "node build-system/check-package-manager.js"
   },
   "dependencies": {
+    "@percy/puppeteer": "0.3.2",
     "babel-polyfill": "6.26.0",
     "document-register-element": "1.5.0",
     "dompurify": "1.0.5",
     "promise-pjs": "1.1.3",
+    "puppeteer": "1.5.0",
```

Comment:
I just realized that these should've been added to `devDependencies`, not `dependencies`